contracts/eosiolib/types.h

```diff
@@ -59,6 +59,8 @@ struct fixed_string16 {
    char str[16];
 };
 
+typedef struct checksum256 transaction_id;
```

Comment:
For consistency, this should match the type name in the non-contract code, so make this "transaction_id_type".